packages/next/src/server/next-server.ts

```diff
@@ -1273,6 +1277,17 @@ export default class NextNodeServer extends BaseServer<
     return handler
   }
 
+  /**
+   * @internal - this method is internal to Next.js and should not be used directly by end-users
+   */
+  public getRequestHandlerWithMetadata(meta: RequestMeta): NodeRequestHandler {
```

Comment:
this API is only used with test mode right? should we no-op it if not enabled?

Reply:
I'll update the comments, but technically [it's used by the builder](https://github.com/vercel/vercel/blob/545dac8ce94eca3f6ce8fb3edece9342637fc046/packages/next/src/server-launcher.ts#L63). Previously, it just used the `base-server.ts` implementation.